patches/server/0073-Add-World-Util-Methods.patch

```diff
@@ -45,7 +65,129 @@ index b75d78da71f24a0b02bf709d026389add0556489..56d9a764bbf521f512e959efaa864c44
      public Chunk getChunkIfLoaded(int x, int z) {
          return ((ChunkProviderServer) this.chunkProvider).getChunkIfLoaded(x, z);
      }
-@@ -652,10 +658,46 @@ public abstract class World implements IBlockAccess {
+@@ -297,7 +303,12 @@ public abstract class World implements IBlockAccess {
+         return this.getType(blockposition).getBlock().getMaterial() == Material.AIR;
+     }
+ 
++    // PandaSpigot start - xyz version of isLoaded
++    public boolean isLoaded(int x, int y, int z) {
++        return getChunkIfLoaded(x >> 4, z >> 4) != null;
++    }
+     public boolean isLoaded(BlockPosition blockposition) {
++    // PandaSpigot stop
+         return getChunkIfLoaded(blockposition.getX() >> 4, blockposition.getZ() >> 4) != null; // PandaSpigot
+     }
+ 
+@@ -362,25 +373,92 @@ public abstract class World implements IBlockAccess {
+         return this.chunkProvider.getOrCreateChunk(i, j);
+     }
+ 
+-    public boolean setTypeAndData(BlockPosition blockposition, IBlockData iblockdata, int i) {
++
++    // PandaSpigot start - add captured versions
```

Comment:
It was particularly about these methods that I said about removing, because, I didn't see any use (Maybe there is any but I've already marked many patches as seen)